### ISSUE TITLE
Authenticate to perform admin tasks

### ISSUE THREAD

**Issue by codygarver** [elementary/files#165]
If I try to move, rename, paste, etc into folders I don't have permission to mess with, Marlin should ask me to authenticate and then perform those tasks.

Bounty: https://www.bountysource.com/issues/967985-authenticate-to-perform-admin-tasks

**Comment by ypoz** [comment]
The support for admin:// backend was added in gvfs 1.29.4 more than a year ago, but there's not much one can (or should) do until Elementary is itself upgraded to a newer Ubuntu release.

**Comment by jeremypw** [comment]
See also https://github.com/elementary/code/issues/679

**Comment by jeremypw** [comment]
@elementary/ux Should the privilege elevation be temporary and return to normal after the operation is carried out?

**Comment by cassidyjames** [comment]
@jeremypw yeah, and ideally it would be scoped to that tab as well to not mess up other file permissions or anything. Maybe we could distinguish that tab to show it has admin permissions?

**Comment by nobodywasishere** [comment]
For copying/moving a file it should be a one-time action.

For opening a folder, it should open up a new window (or a new tab with SUDO at the top or something similar to that)

**Comment by SephReed** [comment]
I had this same issue:  https://github.com/elementary/files/issues/2112

Just wanted to throw in my 2 cents that: It would be really nice to make it easier to access files and such.
